InvenTree/part/forms.py

```diff
@@ -66,33 +47,3 @@ class Meta:
         fields = [
             'quantity',
         ]
-
-
-class EditPartSalePriceBreakForm(HelperForm):
-    """Form for creating / editing a sale price for a part."""
-
-    quantity = RoundingDecimalFormField(max_digits=10, decimal_places=5, label=_('Quantity'))
-
-    class Meta:
-        """Metaclass defines fields for this form"""
-        model = PartSellPriceBreak
-        fields = [
-            'part',
-            'quantity',
-            'price',
-        ]
-
-
-class EditPartInternalPriceBreakForm(HelperForm):
-    """Form for creating / editing a internal price for a part."""
-
-    quantity = RoundingDecimalFormField(max_digits=10, decimal_places=5, label=_('Quantity'))
-
```

Comment:
Question: Is this intended?

Reply:
Yes, these forms have long been unused, price breaks are adjusted using the API. This is just old dead code.